hare/flows_test.go

```diff
@@ -127,48 +122,12 @@ func (m *p2pManipulator) Publish(ctx context.Context, protocol string, payload [
 	return nil
 }
 
-// Test - runs a single CP for more than one iteration.
-func Test_consensusIterations(t *testing.T) {
```

Comment:
Why move this test?

Reply:
i moved it to a consensus_test.go and renamed it to `TestConsensus_MultipleIterations`. tests there are more similar to each other.

Reply:
No preference just wondering